server/src/main/java/org/candlepin/audit/EventFactory.java

```diff
@@ -210,14 +214,25 @@ public Event subscriptionExpired(Subscription subscription) {
 
     public Event complianceCreated(Consumer consumer,
         Set<Entitlement> entitlements, ComplianceStatus compliance) {
+        Map<String, String> eventData = new HashMap<String, String>();
+        eventData.put("consumer_uuid", consumer.getUuid());
+        eventData.put("status", compliance.getStatus());
+        String eventDataJson = "";
+        try {
+            eventDataJson = mapper.writeValueAsString(eventData);
```

Comment:
This is somewhat pedantic, but structurally there is no reason to have eventDataJson exist outside of the try/catch block, since this is an all-or-nothing style of operation.

```
try {
    String eventDataJson = mapper.writeValueAsString(eventData);

    return new Event(...);
}
catch (JsonProcessingException e) {
    ...
}
```